src/components/common/EditApproversList.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div class="list approvers-list">
```

Comment:
The functionality is different, mainly for saving the approvers list at the end of the external share, but also in the future I would want them to look different, since these are different operations.